pkgs/development/web/bun/default.nix

```diff
@@ -94,5 +94,8 @@ stdenvNoCC.mkDerivation rec {
     ];
     maintainers = with maintainers; [ DAlperin jk thilobillerbeck cdmistman coffeeispower ];
     platforms = builtins.attrNames passthru.sources;
+    # Broken for Musl at 2023-01-13, tracking issue:
```

Comment:
The year is 2024 not 2023 :see_no_evil:  ;)

Reply:
Thanks! I have just fixed it.